test/functional/change_stream.test.js

```diff
@@ -201,7 +285,7 @@ describe('Change Streams', function() {
     }
   );
 
-  it('Should support creating multiple simultaneous Change Streams', {
+  it.skip('Should support creating multiple simultaneous Change Streams', {
```

Comment:
should we be skipping this?

Reply:
No I think I skipped this temporarily and it slipped through - I'll re-enable.

Reply:
Done, I also re-enabled a few other tests which were already skipped (back in [this PR](https://github.com/mongodb/node-mongodb-native/commit/2cb48941ac8f46eb15431f9a6dadfdd3f111dee0)) but are passing. 

I'm not sure why they were skipped in originally, but it seems like we might as well re-enable them if they pass.